docs/internal/investigations/dapr-remote-runtime-configuration/configuration_concept.md

```diff
@@ -0,0 +1,25 @@
+# Passing configuration to runtimes concept
+
+## Overview
+
+This document describes the concept of passing a configuration to runtimes from the central point with the usage of [DAPR](https://dapr.io).
+
+![Concept image](assets/concept.png?raw=true "Concept")
+
+## Idea
+
+The main idea is to have a component in Kyma Control Plane (Runtime Director on image), which would be a point of communication with Runtimes through Agents and provide a configuration for them. The configuration would be for example an URL and credentials to some external service (e.g. logging service). After each runtime fetch the new configuration, the DAPR bindings should be updated and populated to the DAPR sidecars.
+
+## Example
```

Comment:
This example fits more to the actual POC documentation, describing the actual use-case to implement.